spec/ruby-mpd/parser_spec.rb

```diff
@@ -89,9 +89,9 @@
     end
 
     context "with :db_update key" do
-      expected_time = Time.local(2015, 6, 11, 21, 14, 33)
-      it { expect(subject.send(:parse_key, :db_update, '1434024873'))
-        .to eql(expected_time) }
+      expected_time = Time.at(1434024873)
+      it { expect(subject.send(:parse_key, :db_update, '1434024873').to_i)
```

Comment:
Yeah I think casting them to `.utc` might be better, but either way works I guess.